common/djangoapps/django_comment_common/utils.py

```diff
@@ -1,9 +1,17 @@
 """
 Common comment client utility functions.
 """
-
 from django_comment_common.models import Role, FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR, \
     FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_STUDENT
+from request_cache.middleware import request_cached
+
+from .models import CourseDiscussionSettings
+from openedx.core.djangoapps.course_groups.cohorts import get_legacy_discussion_settings
+
+
+NOT_DIVIDED = CourseDiscussionSettings.NONE
```

Comment:
@staubina What I meant was that everyone goes through this file already to use get_course_discussion_settings/set_course_discussion_settings. But if you use CourseDiscusssionSettings.COHORT, etc., callers then have to import that (which lives in a different file) in addition to get_course_discussion_settings/set_course_discussion_settings.

I was trying to reduce the number of imports required by callers of these methods, but I don't feel strongly about it and can revert back to the original constants.